index.rst

```diff
@@ -180,14 +180,20 @@ Key resources
 -------------
 
 * Coding style guides
-    * :PEP:`7` (Style Guide for C Code)
-    * :PEP:`8` (Style Guide for Python Code)
+
+  * :PEP:`7` (Style Guide for C Code)
```

Comment:
Should these be three-space indents?

Reply:
Lists work a bit differently (and that's the reason why people often get them wrong).  See
https://devguide.python.org/documentation/markup/#lists-and-quotes

We could clarify these exceptions in the devguide (possibly in both places).

Reply:
For posterity: Using three-space indents in this specific spot would result in a `blockquote` (or `dl`, depending on the absence of a blank line) in this specific spot.